src/layouts/MyLayout.vue

```diff
@@ -4,110 +4,116 @@
       elevated
     >
       <q-toolbar>
+        <q-toolbar-title>
+          <font
+            face="impact"
+            size="6"
+          >
+            ICEMEME
+          </font>
+        </q-toolbar-title>
+
+        <q-btn
+          color="secondary"
+          :to="{ name: 'register' }"
+        >
+          Sign Up / Log In
+        </q-btn>
+
         <q-btn
           flat
           dense
           round
           aria-label="Menu"
-          @click="leftDrawerOpen = !leftDrawerOpen"
+          @click="rightDrawerOpen = !rightDrawerOpen"
         >
           <q-icon name="menu" />
         </q-btn>
-
-        <q-toolbar-title>
-          Quasar App
-        </q-toolbar-title>
-
-        <div>Quasar v{{ $q.version }}</div>
       </q-toolbar>
     </q-header>
 
     <q-drawer
-      v-model="leftDrawerOpen"
+      v-model="rightDrawerOpen"
+      side="right"
       bordered
       content-class="bg-grey-2"
     >
       <q-list>
         <q-item-label header>
-          Essential Links
+          [USERNAME]
         </q-item-label>
         <q-item
           clickable
           tag="a"
-          target="_blank"
-          href="http://v1.quasar-framework.org"
+          :to="{ name: 'home' }"
         >
           <q-item-section avatar>
-            <q-icon name="school" />
+            <q-icon name="home" />
           </q-item-section>
           <q-item-section>
-            <q-item-label>Docs</q-item-label>
+            <q-item-label>Home</q-item-label>
```

Comment:
Maybe instead of "home" this could say "latest" and use the `whatshot` icon? Just a suggestion, what do you think @adnanrh?